requirements.in

```diff
@@ -1,6 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-ops
+ops>2
 oci-image
-serialized-data-interface<0.7
+serialized-data-interface==0.6.0
```

Comment:
Is there a reason why we'd pin this specifically?  Does <0.7 give troubles?

Reply:
I ask mainly because we've been bitten by having such strict pinning in the past